<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="73407" started-at="2014-09-07T11:15:58Z" finished-at="2014-09-07T11:17:12Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="73407" started-at="2014-09-07T11:15:58Z" finished-at="2014-09-07T11:17:12Z">
      <class name="com.example.tests.ContactModificationTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.example.tests.ContactModificationTest@37db1db7]" name="setUp" is-config="true" duration-ms="8666" started-at="2014-09-07T14:15:58Z" finished-at="2014-09-07T14:16:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="modifySomeContactViaEdit(com.example.tests.ContactData)[pri:0, instance:com.example.tests.ContactModificationTest@37db1db7]" name="modifySomeContactViaEdit" duration-ms="32396" started-at="2014-09-07T14:16:07Z" data-provider="randomValidContactGenerator" finished-at="2014-09-07T14:16:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContactData [firstName=testName1476596777fngktbwbzqsmlfilwzna, lastName=testName-1428325622ycjcrmxdisyphclbrpxm]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: <[ContactData [firstName=, lastName=testName-1401653762yljnsypojeqxgedtpgdv], ContactData [firstName=testName-1942777357wognrmoqtpuwwkenyscm, lastName=], ContactData [firstName=testName1476596777fngktbwbzqsmlfilwzna, lastName=testName-1428325622ycjcrmxdisyphclbrpxm], ContactData [firstName=testName408169317cqulossbvzdskoyoikjw, lastName=testName1948549946mtgxezexkjxvetfsjzgt]]]]><![CDATA[>
     but: was <[ContactData [firstName=, lastName=testName-1401653762yljnsypojeqxgedtpgdv], ContactData [firstName=testName-1428325622ycjcrmxdisyphclbrpxm, lastName=testName1476596777fngktbwbzqsmlfilwzna], ContactData [firstName=testName-1942777357wognrmoqtpuwwkenyscm, lastName=], ContactData [firstName=testName408169317cqulossbvzdskoyoikjw, lastName=testName1948549946mtgxezexkjxvetfsjzgt]]><![CDATA[]]]]><![CDATA[>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: <[ContactData [firstName=, lastName=testName-1401653762yljnsypojeqxgedtpgdv], ContactData [firstName=testName-1942777357wognrmoqtpuwwkenyscm, lastName=], ContactData [firstName=testName1476596777fngktbwbzqsmlfilwzna, lastName=testName-1428325622ycjcrmxdisyphclbrpxm], ContactData [firstName=testName408169317cqulossbvzdskoyoikjw, lastName=testName1948549946mtgxezexkjxvetfsjzgt]]]]><![CDATA[>
     but: was <[ContactData [firstName=, lastName=testName-1401653762yljnsypojeqxgedtpgdv], ContactData [firstName=testName-1428325622ycjcrmxdisyphclbrpxm, lastName=testName1476596777fngktbwbzqsmlfilwzna], ContactData [firstName=testName-1942777357wognrmoqtpuwwkenyscm, lastName=], ContactData [firstName=testName408169317cqulossbvzdskoyoikjw, lastName=testName1948549946mtgxezexkjxvetfsjzgt]]]]><![CDATA[>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert.assertThat(Assert.java:832)
	at com.example.tests.ContactModificationTest.modifySomeContactViaEdit(ContactModificationTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- modifySomeContactViaEdit -->
        <test-method status="FAIL" signature="modifySomeContactViaEdit(com.example.tests.ContactData)[pri:0, instance:com.example.tests.ContactModificationTest@37db1db7]" name="modifySomeContactViaEdit" duration-ms="32130" started-at="2014-09-07T14:16:39Z" data-provider="randomValidContactGenerator" finished-at="2014-09-07T14:17:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContactData [firstName=testName-273140888ndsjflzzelpdzkvpdtxf, lastName=testName-1440358431auwqfgrqatsyeynkthus]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: <[ContactData [firstName=, lastName=testName-1401653762yljnsypojeqxgedtpgdv], ContactData [firstName=testName-1942777357wognrmoqtpuwwkenyscm, lastName=], ContactData [firstName=testName-273140888ndsjflzzelpdzkvpdtxf, lastName=testName-1440358431auwqfgrqatsyeynkthus], ContactData [firstName=testName408169317cqulossbvzdskoyoikjw, lastName=testName1948549946mtgxezexkjxvetfsjzgt]]]]><![CDATA[>
     but: was <[ContactData [firstName=, lastName=testName-1401653762yljnsypojeqxgedtpgdv], ContactData [firstName=testName-1440358431auwqfgrqatsyeynkthus, lastName=testName-273140888ndsjflzzelpdzkvpdtxf], ContactData [firstName=testName-1942777357wognrmoqtpuwwkenyscm, lastName=], ContactData [firstName=testName408169317cqulossbvzdskoyoikjw, lastName=testName1948549946mtgxezexkjxvetfsjzgt]]><![CDATA[]]]]><![CDATA[>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: <[ContactData [firstName=, lastName=testName-1401653762yljnsypojeqxgedtpgdv], ContactData [firstName=testName-1942777357wognrmoqtpuwwkenyscm, lastName=], ContactData [firstName=testName-273140888ndsjflzzelpdzkvpdtxf, lastName=testName-1440358431auwqfgrqatsyeynkthus], ContactData [firstName=testName408169317cqulossbvzdskoyoikjw, lastName=testName1948549946mtgxezexkjxvetfsjzgt]]]]><![CDATA[>
     but: was <[ContactData [firstName=, lastName=testName-1401653762yljnsypojeqxgedtpgdv], ContactData [firstName=testName-1440358431auwqfgrqatsyeynkthus, lastName=testName-273140888ndsjflzzelpdzkvpdtxf], ContactData [firstName=testName-1942777357wognrmoqtpuwwkenyscm, lastName=], ContactData [firstName=testName408169317cqulossbvzdskoyoikjw, lastName=testName1948549946mtgxezexkjxvetfsjzgt]]]]><![CDATA[>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert.assertThat(Assert.java:832)
	at com.example.tests.ContactModificationTest.modifySomeContactViaEdit(ContactModificationTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- modifySomeContactViaEdit -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.ContactModificationTest@37db1db7]" name="tearDown" is-config="true" duration-ms="196" started-at="2014-09-07T14:17:12Z" finished-at="2014-09-07T14:17:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.ContactModificationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
